Tests.Integration.Presentation.Web/SystemUsage/ItSystemUsageOverviewReadModelsTest.cs

```diff
@@ -76,6 +76,8 @@ public async Task ReadModels_Contain_Correct_Content()
             var linkToDirectoryUrlName = A<string>();
             var riskSupervisionDocumentationUrl = A<string>();
             var riskSupervisionDocumentationUrlName = A<string>();
+            var generalPurpose = A<string>();
```

Comment:
Tilføjet som en del af integrations testen for om værdien kommer med i read model